template.md

```diff
@@ -34,8 +34,8 @@ TBD Hotels nearby (optional)
 
 ## Agenda Topic Rules
 
-1. Proposals looking to advance must be added to the agenda along with necessary review materials 7 days prior to the first day of the meeting.
-   1. Note: any items added after 7 days prior to the start time of the meeting must be denoted with a ⌛️ symbol, which indicates that it is not eligible for stage advancement, and that it must wait until all other items have been addressed.
+1. Proposals looking to advance must be added to the agenda (meaning, a PR opened and/or merged) along with necessary review materials 7 days (168 hours, precisely) prior to the first day of the meeting.
```

Comment:
_I can imagine we should use moment.js at some point to count how many hours left._

`s/'a PR opened and/or merged'/'a PR opened and/or a commit merged'`

this case covers direct commits even if they lack our full appreciation.

Reply:
Good point, will update